packages/graphback-codegen-schema/tests/__snapshots__/GraphQLSchemaCreatorTest.ts.snap

```diff
@@ -109,13 +114,27 @@ type Comment {
   noteComment: Note!
 }
 
-input CommentFilterInput {
+input CommentCreationInput {
+  id: ID
+  title: String!
+  description: String!
+  note_id: ID!
```

Comment:
@craicoverflow This is example of the problem you mentioned on the call. Automatic generation of the opposite side not always make sense. For example here we getting capability to move comment to different note which in real life makes no sense

Reply:
I don't fully understand - without looking into this too much `note_id` appears to be correct?

Reply:
yes. it behaves correctly but it makes no sense for end-user. 
Why would I need ability to move my comment to different Note etc.
That is why more explicit relationships make sense.

Reply:
Yeah explicit relationships do make more sense, but we put explicit because of those rare cases where a model references another in two fields, for example:

```gql
type Message {
  """
  @manyToOne field: 'sentMessages'
  """
  from: User
  """
  @manyToOne field: 'receivedMessages'
  """
  to: User
}
```

We are aware of the need to enable explicit relationships alright.